user-guide/Troubleshooting/Procedures/Investigating_OpenSearch_Issues.md

```diff
@@ -0,0 +1,165 @@
+          at org.opensearch.transport.InboundHandler.lambda$handleException$3(InboundHandler.java:420) ~[opensearch-2.8.0.jar:2.8.0]
+          at org.opensearch.common.util.concurrent.ThreadContext$ContextPreservingRunnable.run(ThreadContext.java:747) ~[opensearch-2.8.0.jar:2.8.0]
+          at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136) ~[?:?]
+          at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635) ~[?:?]
+          at java.lang.Thread.run(Thread.java:833) [?:?]
+  Caused by: org.opensearch.transport.RemoteTransportException: [DataNodeName][166.206.186.147:9300][internal:cluster/coordination/join/validate]
+  Caused by: org.opensearch.cluster.coordination.CoordinationStateRejectedException: join validation on cluster state with a different cluster uuid tNh7sXJeQjuf-RTTfFt7qg than local cluster uuid qVI2q9lMSy-Ot7O9v68d_A, rejecting
+          at org.opensearch.cluster.coordination.JoinHelper.lambda$new$4(JoinHelper.java:219) ~[opensearch-2.8.0.jar:2.8.0]
+          at org.opensearch.indexmanagement.rollup.interceptor.RollupInterceptor$interceptHandler$1.messageReceived(RollupInterceptor.kt:113) ~[?:?]
+          at org.opensearch.performanceanalyzer.transport.PerformanceAnalyzerTransportRequestHandler.messageReceived(PerformanceAnalyzerTransportRequestHandler.java:43) ~[?:?]
+          at org.opensearch.transport.RequestHandlerRegistry.processMessageReceived(RequestHandlerRegistry.java:106) ~[opensearch-2.8.0.jar:2.8.0]
+          at org.opensearch.transport.InboundHandler$RequestHandler.doRun(InboundHandler.java:453) ~[opensearch-2.8.0.jar:2.8.0]
+          at org.opensearch.common.util.concurrent.ThreadContext$ContextPreservingAbstractRunnable.doRun(ThreadContext.java:806) ~[opensearch-2.8.0.jar:2.8.0]
+          at org.opensearch.common.util.concurrent.AbstractRunnable.run(AbstractRunnable.java:52) ~[opensearch-2.8.0.jar:2.8.0]
+          at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136) ~[?:?]
+          at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635) ~[?:?]
+          at java.lang.Thread.run(Thread.java:833) ~[?:?]
+  ```
+
+To fix this issue:
+
+1. Stop OpenSearch on the cluster manager and data nodes.
+
+1. Check the *cluster.initial_master_nodes* and *discovery.seed_hosts* on all nodes for any differences.
+
+1. Go to the data folder specified in the *opensearch.yml* file and delete the folder called *nodes* along with every file in it by using the following command:
+
+   ```bash
+   sudo rm -rf nodes
+   ```
+
+1. Restart OpenSearch on the cluster manager first and then on the data nodes.
+
+> [!IMPORTANT]
+> Deleting the nodes folder will result in loss of data. You should only do so with a new installation of OpenSearch.
+
+## SLSearch.txt logging mentions OpenSearch version is not officially supported
+
```

Comment:
We currently support 1.x and 2.x. Maybe add info in case Opensearch 3.x is released (futureproofing)

Reply:
Doubt if we should list specific versions here... It is supported so. Unless 3.0 would do something different regarding behavior.